v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -81,7 +81,7 @@ case object FrequencyAnalysis extends V2FConstants {
     Set("chromosome", "position", "reference", "alt")
 
   override def variantFieldsToExtract: Set[String] =
-    Set("chromosome", "position", "reference", "alt")
+    Set("id", "chromosome", "position", "reference", "alt")
```

Comment:
This looks like a constant across all the tables. Can we pull it out of this enum and make it a global?

Reply:
It is constant across all 3 that have variants extracted; it wouldn't make a functional difference to add it as a default, but then 2 of the tables would have it as the `variantFieldsToExtract` nonsensically. If that's okay (which can be explained with a comment) then sure.